h2o-app/build.gradle

```diff
@@ -13,6 +13,8 @@ dependencies {
   if (project.hasProperty("doIncludeMojoPipeline") && project.doIncludeMojoPipeline == "true") {
     compile project(":h2o-ext-mojo-pipeline")
   }
+  compile project(":h2o-persist-hdfs")
```

Comment:
do want to remove these from assemblies where they are explicitly added for packaging now that app depends directly?

Reply:
Removed of course.